core/frontend/src/tile/Tile.ts

```diff
@@ -83,6 +83,7 @@ export abstract class Tile {
   protected _contentRange?: ElementAlignedBox3d;
   /** The maximum size in pixels this tile can be drawn. If the size of the tile on screen exceeds this maximum, a higher-resolution tile should be drawn in its place. */
   protected _maximumSize: number;
+  protected _outOfLodRange?: boolean;
```

Comment:
What does `undefined` indicate? If it means the same as `false`, we don't need it.

Reply:
Actually just remove it from `Tile`.